test/test_backend/test_backend.py

```diff
@@ -86,26 +86,6 @@ def test_hierarchify_list_of_parent_kids(self):
         # self.assertEqual(1, 1)  # todo
         pass
 
-    # TODO: test is failing. fix
-    @unittest.skip("Skipping failing/erroring test temporarily.")
-    def test_csets_update(self):
```

Comment:
## Delete: `test_csets_update()`
No longer need this test because we're not using `/datasets/csets/` in this way anymore. This was for our pre-PostgreSQL data layer implementation.

Along with this, I also deleted:
- csets_read(), csets_git_update(), vocab_update(), csets_update()